### PR TITLE
fix a few typos in the doc

### PR DESCRIPTION
**Pull request by tatchi**
Just a few typos I noticed while reading this document

### CODE REVIEW

File: doc/ppx-for-plugin-authors.rst [hunk 1]
```diff
@@ -111,7 +111,7 @@ You will also need the following ``my_ppx_rewriter.ml``:
 
 There are a few things to explain here. The last part, i.e. the call to
 ``Driver.register_transformation`` is common to almost all ppxlib-based PPX plugins and is how
-you let ppxlib know about your transformation. You'll note that here we register a single rule
+you let ``ppxlib`` knows about your transformation. You'll note that here we register a single rule
```
Comment (NathanReb) [inline doc/ppx-for-plugin-authors.rst:114]: This wasn't a typo: 
```suggestion
you let ``ppxlib`` know about your transformation. You'll note that here we register a single rule
```

File: doc/ppx-for-plugin-authors.rst [hunk 2]
```diff
@@ -183,7 +183,7 @@ You will also need the following ``my_ppx_deriver.ml``:
        ~sig_type_decl:intf_generator
 
 
-The call to ``Deriving.add`` is how you'll let ``ppxlib`` know about your deriver. The first string
+The call to ``Deriving.add`` is how you'll let ``ppxlib`` knows about your deriver. The first string
```
Comment (NathanReb) [inline doc/ppx-for-plugin-authors.rst:186]: Same here!
```suggestion
The call to ``Deriving.add`` is how you'll let ``ppxlib`` know about your deriver. The first string
```

**Comment by tatchi** [comment]
Thanks for the review @NathanReb, I did apply your suggested changes.

**Review by NathanReb — APPROVED** [review]
Sorry I forgot to come back to this! Thanks for the fix!

**Pull request merged**